ch.sbb.scion.rcp.microfrontend.e3.app.demo/plugin.xml

```diff
@@ -22,7 +22,7 @@
    <extension
          point="org.eclipse.ui.views">
       <e4view
-            class="ch.sbb.scion.rcp.microfrontend.e3.app.demo.view.MessageClientPart"
+            class="ch.sbb.scion.rcp.microfrontend.e3.appbreup.demo.view.MessageClientPart"

```

Comment:
Is this change intentional? (The package of the MessageClientPart was not changed, btw.)

Reply:
No, absolutely not. No idea where this was coming from. Will check.